### PR TITLE
First page

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request, I have done the following:

-created an html file
-created a CSS file
-parsed my figma design to create my portfolio's first page
-checked and rectified linter errors

Please have a look and get back to me😊

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    </header>
+    <section class="section-1">
+        <h1 class="heading-1">
+            Busiwa Liuma
+        </h1>
+        <p class="par-1">
+        Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button class="button-1" type="button">Start Collaboration</button>
```
Comment (emmanuellekamwa) [inline index.html:29]: - [x] There should be a list of icons on your page and the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Please fix that.
- [x] Please you have to make a professional readme file. For now your portfolio has not been done yet. Try to fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: REQUEST CHANGES 🛑🛑**
Hello @Busiwa24 👋🏽👋🏽
Good job so far!
You have almost passed all the requirements but there are some issues that you still need to work on to go to the next project but you are almost there!
## Required Changes ♻️
_Check the comments under the review._👇🏽👀
Please try as much as possible to stick to the given design. Thanks.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- For your readme file I will suggest that you should add a live demo link and a screenshot to make it more professional.

Cheers and Happy coding! 👏🏽👨🏽‍💻👏🏽

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Busiwa24) [reply index.html:29]: Thanks, it's been handled

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <link href="style.css" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-bar">
+            <a href=""><i class="fa fa-bars hamburger" aria-hidden="true"></i></a>
+        </nav>
+    </header>
+    <section class="section-1">
```
Comment (Berabjesus) [inline index.html:21]: - [ ] [OPTIONAL] I suggest using the `main` tag to wrap the main content of your web page.
```
<main>
    <section></section>
</main>
```

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
#### Hey @Busiwa24 Excellent job on this project. You have implemented all the required changes.
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete. 👍** 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Use the `main` tag for the main section of your page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
I wish you all the very best in your next tasks:1st_place_medal:

**Cheers and Happy coding!👏👏👏**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**